.teamcity/_self/projects/WPComTests.kt

```diff
@@ -188,14 +197,18 @@ fun jetpackPlaywrightBuildType( targetDevice: String, buildUuid: String): E2EBui
 				allowEmpty = false
 			)
 			param("env.VIEWPORT_NAME", "$targetDevice")
-			checkbox(
-				name = "env.TEST_ON_JETPACK",
-				value = "true",
-				label = "Test on Jetpack",
-				description = "Use a Jetpack blog to test against",
-				checked = "true",
-				unchecked = "false"
+			select(
+				name = "env.JETPACK_TARGET",
+				value = jetpackTarget,
+				label = "Jetpack Target",
+				description = "Which Jetpack install are we targeting through Calypso?",
+				options = listOf(
+					"wpcom-production",
+					"wpcom-staging",
+					"not-wpcom"
```

Comment:
Potentially better name could be `remote-site`?

Reply:
Oh, that's much better lololol

Reply:
My faux-enum writing skills never really graduated past the third grade 🤣

Reply:
Fixed in [6acd928](https://github.com/Automattic/wp-calypso/pull/73457/commits/6acd928c6cfc133d195d5578dc07e44821eef125)